ansible_rulebook/rule_set_runner.py

```diff
@@ -128,10 +128,10 @@ async def run_ruleset(self):
             tasks.append(self.source_loop_task)
             await asyncio.wait([self.action_loop_task])
         except asyncio.CancelledError:
-            logger.debug("Cancelled error caught in run_ruleset")
+            logger.error("Cancelled error caught in run_ruleset")
             for task in tasks:
                 if not task.done():
-                    logger.debug("Cancelling (2) task %s", task.get_name())
+                    logger.info("Cancelling (2) task %s", task.get_name())
```

Comment:
@mkanoor great point -- I'm always torn on how to distinguish errors from their handler side effects. I think your idea is good and it's consistent.

I assume it won't result in a lot of extra log messages because async cancellations should be rare, is that true?

Reply:
Omitting this change for now because it exceeds the original scope.